lib/EnzymeCore/src/EnzymeCore.jl

```diff
@@ -61,6 +83,10 @@ accumulator for gradients (so ``\\partial f / \\partial x`` will be *added to*)
 struct Duplicated{T} <: Annotation{T}
     val::T
     dval::T
+    function Duplicated(val::T, dval::T; checked=structure_check) where T
```

Comment:
if the preference isn't there, will this be a static no check, or will it load a global then check?

If the latter, I want to have a way to avoid the performance penalty.

Reply:
It's a global const so it's propagated (and it's a compile time preference)